### ISSUE TITLE
Text typed on tabby only shows after typing 4-5 characters

### ISSUE THREAD

**Issue by TahaCoder43** [Eugeny/tabby#8954]
I have just downloaded tabby terminal, I installed no plugin, did nothing. I just opened it, and i notice that when i type I don't see anything, and that only after typing 4-5 characters tabby renders what I typed in the terminal.

I have tried disabling every feature in tabby but the problem doesn't seem to go away

I tried

- disabling Gpu acceleration
- xterm(webgl) to xterm(canvas)
- turning off Sixel graphics support
- Draw bold text in bright colorus
- font ligratures

Nothing is working, I am on linux with kde neon

**Comment by Tbhesswebber** [comment]
This happened to me upon upgrading to v1.0.200 from v1.0.197 (worked fine in 197).  

Given that 200 had a configuration migration, I don't know if downgrading is a safe route to go, so I'm just dealing with it for now and using a different terminal for any "real" terminal usage

**Comment by TheBlueKnight42** [comment]
Duplicate of https://github.com/Eugeny/tabby/issues/8884
I have not tested with KDE Neon, but I am running on Kubuntu 23.04, which should be very similar. When you switched from xterm(webgl) to xterm(canvas) did you close out the terminal session before testing? I only ask because in my test, if I did not close the session, the problem remained, but if I exited the session and started a new one, it was resolved. I hope this can help.
Blue
